duckduckgo/Android
# Block malware to production

Task/Issue URL: https://app.asana.com/0/1203137811378537/1208346457678774/f

### Description
Refactor block malware US according to wireframes in [here](https://app.asana.com/0/inbox/1198194956790048/1206289678428945/1208360300072258/f).

### Steps to test this PR
* UI according to wireframes
* Repeat test in https://github.com/duckduckgo/Android/pull/4979
* Play builds should not see block malware option
